packages/cli/test/cases/build.config.markdown-custom.settings/greenwood.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  markdown: {
+    settings: { gfm: false }
```

Comment:
oh.. this already supported?  Does that mean this [issue request to **wc-markdown-loader**](https://github.com/hutchgrant/wc-markdown-loader/issues/19) would get resolved?

Reply:
yep

Reply:
good to know!  Will try and test it out.